dev-lang/rust/rust-1.54.0.ebuild

```diff
@@ -105,7 +105,10 @@ DEPEND="
 	net-misc/curl:=[http2,ssl]
 	sys-libs/zlib:=
 	dev-libs/openssl:0=
-	elibc_musl? ( sys-libs/libunwind:= )
+	elibc_musl? (
+		llvm-libunwind? ( sys-libs/llvm-libunwind:= )
```

Comment:
is it really musl-only thing? why is it in this block?

Reply:
It makes sense to me but I'm not entirely sure. Every rust ebuild only depends on libunwind for musl.

The config.toml.example has this comment:

```
# Use LLVM libunwind as the implementation for Rust's unwinder.
# Accepted values are 'in-tree' (formerly true), 'system' or 'no' (formerly false).
# This option only applies for Linux and Fuchsia targets.
# On Linux target, if crt-static is not enabled, 'no' means dynamic link to
# `libgcc_s.so`, 'in-tree' means static link to the in-tree build of llvm libunwind
# and 'system' means dynamic link to `libunwind.so`. If crt-static is enabled,
# the behavior is depend on the libc. On musl target, 'no' and 'in-tree' both
# means static link to the in-tree build of llvm libunwind, and 'system' means
# static link to `libunwind.a` provided by system. Due to the limitation of glibc,
# it must link to `libgcc_eh.a` to get a working output, and this option have no effect.
```

Gentoo never seems to use crt-static, so systems with musl could link to libgcc_s.so and systems without musl could dynamically link to libunwind.so, whether via llvm or not. Though, if crt-static were used, things would get more complicated, since static musl and static glibc never overlap.

My goal was to get rust working with clang/libcxx/llvm-libunwind, which this change enables, though there are other possibilities.

Suggestions welcome.